images/server/install-packages.sh

```diff
@@ -6,24 +6,34 @@ get_custom_repo() {
     url="$1"
     fname="$(basename "$url")"
     dest="/etc/yum.repos.d/${fname}"
+    dnf install --setopt=install_weak_deps=False -y curl
```

Comment:
I think it would be good to squash this entire patch with the previous patch. It would "undo" some of the unwanted changes that I previously commented on.

Reply:
@phlogistonjohn if I see it right, the previous patch is the one that adds the capability to explicitly choose  centos and fedora as base OS, right? Shouldn't I rather squash with the commit that introduces the script in the first place?